src/snapshots/003.txt

```diff
@@ -0,0 +1,24 @@
+~~~META
+description=Import statment
+~~~SOURCE
+module [decoder]
+
+import json.Json
+
+decoder = Utf8.decode
+~~~FORMATTED
+module [decoder]
+
+import json.Json
+
+decoder = Utf8.decode
+~~~PARSE
+(file
+    (header 'decoder')
+    (import
+        'json'
+        '.Json'
+        '')
```

Comment:
What's up with the empty string at the end there? That seems wrong...

Reply:
I would do an optional sexpr term at the end, possibly wrapped in (exposed (whatever)) or something for clarity

Reply:
I'll look at this in a follow up PR I think. I like the idea of an optional child node for exposed.